source/mod_fluka.f90

```diff
@@ -1099,9 +1099,9 @@ subroutine kernel_fluka_element( nturn, i, ix )
            napx, xv1, xv2, yv1, yv2, sigmv, ejv, naa, nzz, nucm, nqq, pdgid, &
            spin_x, spin_y, spin_z )
 
-      if (ret.eq.-1) then
-         write(lerr,'(A)')'FLUKA> ERROR -1 in Fluka communication returned by fluka_send_receive...'
-         write(fluka_log_unit,'(A)')'# Error -1 in Fluka communication returned by fluka_send_receive...'
+      if (ret.lt.0) then
+         write(lerr,'(A,i4,A)')'FLUKA> ERROR ', ret, ' in Fluka communication returned by fluka_send_receive...'
```

Comment:
Why not use format `i0` so there is no chance of `****`?

Reply:
I agree with you. @vkbo - even though I don't think we will have many error codes, but it is better to set it once correctly forever.